src/graphics/text.rs

```diff
@@ -1,11 +1,11 @@
 use super::*;
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq)]
-pub struct Font {}
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Removing Copy from the Font break API-compatibility with ggez. And this specific Copy trait is used by [Zemeroth](http://github.com/ozkriff/zemeroth) a lot :( 

So we cant remove Copy from the Font. 

Probably we need ggez-like FontId and store some FontId->String HashMap in graphics context? Not sure :(

Reply:
Oh, interesting. We'll take a look at the ggez source and see how that works. Thanks!

Reply:
On the other hand, ```graphics::Font("small-caps 10px Courier New".to_string())``` definetly looks good and make sense for the canvas api. 

I am not sure yet, but I think that ggez-style text should renders throw webgl with custom ttf's and be 100% compatible with the ggez api. 
While its ok to have additional CanvasText/CanvasFont for canvas-only text rendering. 

Still thinking about it though, any ideas welcome! :)